doc/query.md

```diff
@@ -0,0 +1,545 @@
+# Cloudant Query - Android
+
+Cloudant Query - Android is a Java implementation of [Cloudant Query][1] for Android mobile devices. It is included as a sub-package of [sync-android][2].
```

Comment:
Updated in https://github.com/cloudant/sync-android/commit/d5b4f7825035187c3ef55b5962f38b66806f68d3